contracts/truefi2/TrueFiPool2.sol

```diff
@@ -0,0 +1,414 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+
+import {ERC20} from "../common/UpgradeableERC20.sol";
+import {Ownable} from "../common/UpgradeableOwnable.sol";
+
+import {ITrueStrategy} from "./interface/ITrueStrategy.sol";
+import {ITrueFiPool2} from "./interface/ITrueFiPool2.sol";
+import {ITrueLender} from "./interface/ITrueLender.sol";
+import {ABDKMath64x64} from "../truefi/Log.sol";
+
+/**
+ * @title TrueFi Pool
```

Comment:
generally update documentation for the contract and functions to be consistent with the new pool design.